javascript/javascript1/week1/house.js

```diff
@@ -0,0 +1,27 @@
+let firstHouseWidth = 8;
+let firstHouseDepth = 10;
+let firstHouseHeight = 10;
+let gardenSizeInM2 = 100;
+let volumeInMeters;
+let sellingPrice_Peter = 2500000;
+let sellingPrice_Julia = 1000000;
+let secondHouseWidth = 5;
+let secondHouseDepth = 11;
+let secondHouseHeight = 8;
+let secondGardenSizeInM2 = 70;
+volumeInMeters = firstHouseDepth * firstHouseWidth * firstHouseHeight ;
```

Comment:
I would also recommend that you add space between some lines to make your code easier to read. Group variables at the top like you did is great, however it's easier for others to review when you provide space between different types of statements such as re-assigning a variable or creating a conditional.